### PR TITLE
Support space level statistics

### PR DESCRIPTION
**Pull request by panda-sheep**
Get the statistics of the current space through the following syntax：

```
submit job statis
show status,
```

### CODE REVIEW

**Comment by Amber1990Zhang** [comment]
```
SUBMIT JOB STATISTIC
```
Is this a typo?

**Review by liuyu85cn — DISMISSED** [review]
LGTM

File: src/common/clients/meta/MetaClient.cpp [hunk 1]
```diff
@@ -3080,5 +3080,22 @@ MetaClient::listGroups() {
     return future;
 }
 
+folly::Future<StatusOr<cpp2::StatisItem>>
+MetaClient::getStatus(GraphSpaceID spaceId) {
```
Comment (critical27) [inline src/common/clients/meta/MetaClient.cpp:3084]: Better to rename to somthing like getStatistic, getStatus could be misleading.

**Comment by panda-sheep** [comment]
> ```
> SUBMIT JOB STATISTIC
> ```
> 
> Is this a typo?

no

File: src/common/clients/meta/MetaClient.cpp (hunk 1 above)
Comment (panda-sheep) [reply src/common/clients/meta/MetaClient.cpp]: Good point,  use `getStatis`.

**Review by critical27 — APPROVED** [review]

**Review by liuyu85cn — APPROVED** [review]
LGTM

**Comment by bright-starry-sky** [comment]
Is it related to any other PR? can merge ?

**Review by bright-starry-sky — APPROVED** [review]

**Comment by panda-sheep** [comment]
> can merge ?
let's go！

**Pull request merged**